0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -141,12 +182,18 @@ The response body is a JSON object that includes basic account details necessary
-| `balance`  | Object | _(OPTIONAL)_ Details of this receiver's balance. Used for invoices and similar temporary accounts. |
-| `balance.maximum` | Integer String | Maximum amount, denoted in the minimum divisible units of the receiver's account, which the receiver will accept. This represents the highest sum that incoming chunks are allowed to reach, not the highest size of an individual chunk (which is determined by path MTU). If this is an invoice the `balance.maximum` is the amount at which the invoice would be considered paid. |
+| `balance`  | Object | _(OPTIONAL)_ Details of this server account's balance. Used for invoices and similar temporary accounts. |
+| `balance.maximum` | Integer String | Maximum amount, denoted in the minimum divisible units of the server's account, which the server will accept. This represents the highest sum that incoming chunks are allowed to reach, not the highest size of an individual chunk (which is determined by path MTU). If this is an invoice the `balance.maximum` is the amount at which the invoice would be considered paid. |
 | `balance.current` | Integer String | Current sum of all incoming chunks. |
-| `asset_info` | Object | _(OPTIONAL)_ Details about the destination asset, for sender's display purposes. |
-| `asset_info.code` | String | Asset code to identify the receiver's currency. Currencies that have [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codes should use those. Sender UIs SHOULD be able to render non-standard codes |
-| `asset_info.scale` | Integer | The scale of the amounts on the receiver's account (e.g. an amount of `"1000"` with a scale of `2` translates to `10.00` units of the receiver's asset/currency) |
+| `pull_balance` | Object | _(OPTIONAL)_ Details of this pull payment token. Used only for pull payment accounts. |
+| `pull_balance.maximum_amount` | Integer String | Maximum amount, denoted in the minimum divisible units of the server's account, which can be pulled from the server. This represents the highest sum that outgoing chunks are allowed to reach, not the highest size of an individual chunk (which is determined by `pull_balance.refill_amount`). |
+| `pull_balance.current_amount` | Integer String | Amount, denoted in the minimum divisible units of the server's account, that can currently be pulled from the account. |
+| `pull_balance.refill_amount` | Integer String | Amount, denoted in the minimum divisible units of the server's account, that will be added to `pull_balance.current_amount` whenever now is later than `pull_balance.refill_date`. |
+| `pull_balance.refill_date` | String | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp in UTC representing the date after which the next pull payment of maximum `pull_balance.refill_amount` can be made, given that `pull_balance.maximum_amount` has not been reached yet. On setting, it has to be in the future. |
```

Comment:
I was thinking about date and time, so the correct name for it would probably be `refill_time`. I don't see though that this suggests that one can only pull on one specific date.